fonts/README.md

```diff
@@ -0,0 +1,27 @@
+#### cloudify
```

Comment:
Why is it duplicate?

Reply:
It's ok, I meant why is this also in  docs/README.md ?

Reply:
Yeah, got it. My intention was to have documentation auto-generated. The file you are talking about (`docs/README.md`) is generated with [documentation.js](http://documentation.js.org/) it takes .md files and merges them according to configuration (`docs/config.yml`) as well as it generates JS code documentation.

I originally thought of exposing the whole documentation somewhere else (GitHub wiki or CFY S3 - there's an option to generate HTML instead of MD), but ended up with simpler solution (don't want to spend too much time on that repo). With original solution I think it would make sense, but right now... Your question makes me think it is not right, I'll change that. I'll only keep auto-generated JS documentation and make references to static assets README.md files in the main README.md file. That way it will not be duplicated.

Reply:
Who do u think will read it? customers? because of we, for example, the rest client docs that are auto gen

Reply:
That's only internal at the moment. It will not be used outside Cloudify.

For JavaScript we're using [ESDoc](http://esdoc.org) - eg. for basic component - https://docs.cloudify.co/widgets-api/latest/
but it is not maintained well and I think we'll need to switch out of it. [documentation.js](http://documentation.js.org/) looks good and is better maintained. Both are dedicated to JS code and bases on JSDoc syntax.

I did a research yesterday and today and didn't find a really good alternatives with quick setup for JS code.

I think after the latest change (removed duplication) it's fine and enough for internal purposes.